requirements.sh

```diff
@@ -28,13 +28,14 @@ os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
 os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
 echo "Platform is ${os_name}, Version: ${os_version}"
 
-if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"9"* ]]; then
 	if [[ $os_name == *"Red Hat"* ]]; then
 		sudo yum-config-manager --enable 'Red Hat Enterprise Linux Server 7 RHSCL (RPMs)'
 		sudo yum install -y @development
 	else
 		sudo yum groupinstall "Development tools" -y
-		sudo yum install -y centos-release-scl
+		# Commented installation of centos-release-scl because its package is not available for Centos Stream 9
```

Comment:
As above - I would be tempted to have separate branches for CentOS 7 and CentOS S9 or make the scl stuff dependant on CentOS7